salt/docker/files/daemon-logging.json

```diff
@@ -0,0 +1,7 @@
+{
+  "log-driver": "syslog",
+  "log-opts": {
+    "syslog-address": "unixgram:///var/spool/docker-ocp.sock",
+    "tag": "docker-ocp-{{.ImageName}}"
```

Comment:
`{{.ImageName}}` is `ghcr.io/open-contracting/kingfisher-process-django:latest`.
In hindsight, `{{.ImageName}}` risks containers sharing log files even if they have different purposes, `{{ .Name }}` should resolve this.

The risk with `{{.Name}}`, and why I chose `{{.ImageName}}`, is because when the container name is not configured then Docker assigns a random name ("objective_wilbur", "recursing_wescoff" etc). Obviously a random name will make log files difficult to identify.
We just need to ensure that any containers created have sensible names, this should be done already in Docker compose. The only other risk is one-off containers made by hand.

I have just changed this to `{{.Name}}`.